reviewdog.go

```diff
@@ -11,6 +11,9 @@ import (
 	"github.com/haya14busa/reviewdog/diff"
 )
 
+var ExportVar = 1
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported var ExportVar should have comment or be unexported